matrix-guide.md

```diff
@@ -45,6 +45,57 @@
+
+## How To...
+
+(These "how to"s apply if you are using the Element client, but other Matrix clients have similar ways to accomplish the same thing.)
+
+### ...see who is in the channel?
+
+Click on "Room Info" (the 🛈 circled-**i** icon in the top right hand corner). Note that not all homeservers expose online/offline info to everyone, so someone may be online even if they are marked as offline.
+
+### ...DM someone?
+
+Click on their avatar in front of one of their messages, or click on their avatar in the list in "Room Info" (the 🛈 circled-**i** icon in the top right hand corner). Then click "Direct Message" (in the menu near the bottom of the right hand panel) to start a private one-to-one room.
+
+To do this IRC-style, you can also type `/msg jsfan hi, what's up?` to start a DM room (in this case with the user `jsfan`).
+
+### ...start a chat with a small group?
+
+Click on the **+** (plus sign) next to "Rooms" (in the room list on the left sidebar).
+You can choose to make it public (discoverable in the public room list) or private (invite-only).
+End-to-end encryption is probably fine to leave on.
+
+### ...use formatting in messages?
+
+Clicking the smiley face icon at the bottom left, or typing <kbd>:</kbd> brings up the emoji picker.
+At the moment the emoji picker is slightly clunky in Element, unfortunately, compared to other chat clients.
+
+You can use Markdown formatting in your messages: `*` for bold, `_` for italics, backticks for code.
+You can even do code blocks with syntax highlighting, using triple backticks; to add a newline in the message without sending it, use <kbd>Shift</kbd>+<kbd>Enter</kbd>.
+(That is, <kbd>&grave;</kbd>, <kbd>&grave;</kbd>, <kbd>&grave;</kbd>, <kbd>J</kbd>, <kbd>S</kbd>, <kbd>Shift</kbd>+<kbd>Enter</kbd>, <kbd>Ctrl</kbd>+<kbd>V</kbd> to paste your code, <kbd>Shift</kbd>+<kbd>Enter</kbd>, <kbd>&grave;</kbd>, <kbd>&grave;</kbd>, <kbd>&grave;</kbd>, <kbd>Enter</kbd>).
+
+Unordered and ordered lists also work the same way as they do in markdown, so <kbd>Shift</kbd>+<kbd>Enter</kbd> comes in handy there as well.
+
+### ...correct mistakes in messages?
+
+You can edit your own messages if you make a typo. To do this, hover over the message and click the pencil icon.
+
+As a shortcut, to edit your most recent message, you can press the up arrow <kbd>↑</kbd>.
+
+You can also remove your message if you post it by mistake (for example, in the wrong channel). To do this, hover over the message and click the **…** (ellipsis), then click "Remove". A "Message deleted" will show up in its place.
+
+### ...block someone?
+
+To block someone so that you don't see their messages, click on their avatar in front of one of their messages, or click on their avatar in the list in "Room Info" (the 🛈 circled-**i** icon in the top right hand corner). Then click "Ignore" (in red at the bottom of the menu in the right hand panel).
```

Comment:
we should probably not be encouraging delegates to block other delegates; anything that warrants that should be surfaced to the CoC committee.

Reply:
I didn't interpret this as blocking other delegates, but blocking other people, maybe spamming, etc.  🤷

Reply:
matrix is an open platform, if we're telling people to use it, we should explain how to prevent interaction with toxic users.

Reply:
Totally agree. Can we be clear that delegates should not block other delegates, and should instead reach out to the chairs and/or the CoC committee?

Reply:
I think we should let people decide if they want to ignore another user. What's the harm in doing that without precluding the possibility of a CoC report?

Reply:
It impedes the technical communication we're all here to do. This is like a workplace - and in a workplace you wouldn't set your colleagues on "ignore".

Reply:
I have changed the language in the heading to say "mute" instead of "block" and to make clear that this is for toxic behaviour, not just for disagreeing with someone. I've also added a paragraph on making a code of conduct report on harassment. I hope this addresses the concerns.

> It impedes the technical communication we're all here to do. This is like a workplace - and in a workplace you wouldn't set your colleagues on "ignore".

Sorry, but I disagree 100% with this. In this hypothetical case, it's the _harassment_ that is impeding the technical communication we're all here to do, not the muting. If a coworker was harassing me on workplace chat, I'd report them to HR, but I'd _also_ put them on ignore.

Reply:
I completely agree that the harassment is impeding the communication in that scenario - but the way "muting" features tend to be used is to silence somebody without such a report (including, for reasons that wouldn't warrant one), and I personally think that muting someone, absent toxic (and thus, reportable) behavior, is a hostile act that I'd prefer not to encourage other delegates to take.